### ISSUE TITLE
(Android) Notification flow not matching documentation

### ISSUE THREAD

**Issue by NoelDeMartin** [arnesson/cordova-plugin-firebase#82]
According to the documentation:

> ```
> App is in foreground:
>     User receives the notification data in the JavaScript callback without any notification on the device itself (this is the normal behaviour of push notifications, it is up to you, the developer, to notify the user)
> App is in background:
>     User receives the notification message in its device notification bar
>     User taps the notification and the app opens
>     User receives the notification data in the JavaScript callback
> ```

However, looking at the code for android two notes can be found:

```
FirebasePluginMessagingService.java line 69:
// TODO: Add option to developer to configure if show notification when app on foreground

FirebasePlugin.java line 125:
// TODO: send buffered notifications here. see iOS implementation
```

I would remove this from the documentation (or change it) until this is developed, since it caused me a couple of hours of confusion + debugging.

My current scenario is to receive a notification from the server when the app is in the foreground, and this cannot be achieved with the current status of the plugin.

**Comment by robertarnesson** [comment]
thx a lot for this. changes are included in 0.1.18

**Comment by NoelDeMartin** [comment]
thank you for the plugin 😄
